src/testing/flow_solver.h

```diff
@@ -41,29 +61,17 @@ class FlowSolver : public TestsBase
     const Parameters::AllParameters all_param; ///< All parameters
     const Parameters::FlowSolverParam flow_solver_param; ///< Flow solver parameters
     const Parameters::ODESolverParam ode_param; ///< ODE solver parameters
-    const double courant_friedrich_lewy_number; ///< Courant-Friedrich-Lewy (CFL) number
     const unsigned int poly_degree; ///< Polynomial order
     const double final_time; ///< Final time of solution
     const std::string unsteady_data_table_filename_with_extension; ///< Filename (with extension) for the unsteady data table
```

Comment:
This should be moved to `periodic_cube_flow`

Reply:
Fixed in recent commit